requirements/edx/base.txt

```diff
@@ -769,7 +769,7 @@ multidict==6.1.0
     #   yarl
 mysqlclient==2.2.4
     # via -r requirements/edx/kernel.in
-newrelic==9.13.0
+newrelic==10.1.0
```

Comment:
No breaking change mentioned in the release notes. [[CHANGELOG](https://github.com/newrelic/newrelic-python-agent/releases/tag/v10.2.0)]